lib/config/presets/internal/replacements.ts

```diff
@@ -587,4 +587,48 @@ const mui: PresetTemplate = {
   title: 'material-ui-to-mui',
 };
 
-addPresets(presets, mui);
+const messageFormat: PresetTemplate[] = [
+  {
+    description:
```

Comment:
please use one preset with multiple package rules

Reply:
this is currently isnt supported by the `addPresets` function, i think ill open a new refactor PR for it, and add the replacements afterwards.
wdyt?

Reply:
why not simply add to static list above?
i don't see a big benefit of abstracting the present generation 🤔

Reply:
it saves a bit of code, in this case it isnt much i guess
e.g. on the left it is the abstracted preset while on the right is the auto generated replacements.

in this case it is 3(auto) vs 5(gen), but in `mui`'s case it was 1vs12 if i recall correctly.

![image](https://user-images.githubusercontent.com/97394622/194029697-42958179-9500-469a-91d7-bcaf3e6a1e29.png)